### PR TITLE
Backport: New install app payload

### PR DESCRIPTION
**Pull request by matthme**

### CODE REVIEW

File: test/e2e/index.ts [hunk 1]
```diff
@@ -1548,7 +1548,7 @@ test(
   })
 );
 
-test.only(
+test(
```
Comment (matthme) [inline test/e2e/index.ts:1549]: I'm assuming that this was accidental...

**Comment by jost-s** [comment]
I've fixed the tests now that DPKI is no longer present.

File: Cargo.toml [hunk 2]
```diff
@@ -7,4 +7,4 @@ resolver = "2"
 opt-level = "z"
 
 [workspace.dependencies]
-hdk = "0.4.0-rc.0"
+hdk = "0.4.0-rc.1"
```
Comment (jost-s) [inline Cargo.toml]: please revert this

**Comment by matthme** [comment]
Looks like tests passed - is it approvable now? =D

**Review by jost-s — CHANGES_REQUESTED** [review]
One change please

File: Cargo.toml (hunk 2 above)
Comment (matthme) [reply Cargo.toml]: It was actually necessary due to the __hc_preflight_request function I think.

Comment (matthme) [reply Cargo.toml]: Tests would fail otherwise

Comment (matthme) [reply Cargo.toml]: Hmm...I cannot reproduce it anymore. I reverted it. But I did get that error at some point and thought I had to bump the hdk for it to work...

**Review by jost-s — APPROVED** [review]

**Pull request merged**